src/components/OrgNode.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="org-node">
+    <input :id="`${prefix}-${id}`" type="checkbox">
```

Comment:
Maybe this should move inside the `li`, because `ul` can only contain `li`s as children?

Reply:
My bad, the `div` wrapping this would need to move inside the `li`